lib/serverengine/multi_spawn_server.rb

```diff
@@ -69,6 +69,10 @@ def reload_config
       nil
     end
 
+    def spawn(process_manager)
```

Comment:
I meant that this method is not intended for override. No subclasses override this method.